src/components/PreviewContent.tsx

```diff
@@ -80,7 +80,7 @@ export const PreviewContent = ({
         <Tabs
           value={activeTab}
           onValueChange={setActiveTab}
-          className="flex-grow flex flex-col"
+          className="flex-grow flex flex-col sticky top-0 h-screen"
```

Comment:
hmm is this needed? the container already has sticky

Reply:
Yes, that container sticky even when provided with the `h-screen` it doesn't make it stick. I think we can remove that div as a whole

https://github.com/user-attachments/assets/75a88b52-774e-46fd-92e9-41881bfdad8f

Reply:
it might be my computer but the videos you're posting are 0s, is it just me?

Reply:
I still don't understand what this line fixes though, it's already in a sticky container. Maybe the video would help (let me know if it's on my end)

Reply:
I think on my end, the video recorded was playing in my machine. But on the upload preview it doesn't

Reply:
Very strange. I use [Kap](https://getkap.co/) for recording fwiw